Parkinson/Startup/APHAppDelegate.m

```diff
@@ -716,40 +678,36 @@ - (ORKTaskViewController *)consentViewController
                                                            propertiesFileName:kConsentPropertiesFileName];
     ORKTaskViewController*  consentVC = [[ORKTaskViewController alloc] initWithTask:task
                                                                         taskRunUUID:[NSUUID UUID]];
-    
     return consentVC;
 }
 
-- (NSDictionary *) tasksAndSchedulesWillBeLoaded {
-    
-    NSString                    *resource = [[NSBundle mainBundle] pathForResource:self.initializationOptions[kTasksAndSchedulesJSONFileNameKey]
+- (NSDictionary *)tasksAndSchedulesWillBeLoaded
+{
+    NSString      *resource = [[NSBundle mainBundle] pathForResource:self.initializationOptions[kTasksAndSchedulesJSONFileNameKey]
                                                                             ofType:@"json"];
-    
-    NSData                      *jsonData = [NSData dataWithContentsOfFile:resource];
-    NSError                     *error;
-    NSDictionary                *dictionary = [NSJSONSerialization JSONObjectWithData:jsonData
+
+    NSData        *jsonData = [NSData dataWithContentsOfFile:resource];
+    NSError       *error;
+    NSDictionary  *dictionary = [NSJSONSerialization JSONObjectWithData:jsonData
                                                                               options:NSJSONReadingMutableContainers
                                                                                 error:&error];
     if (dictionary == nil) {
         APCLogError2 (error);
     }
-    
-    NSArray                     *schedules = [dictionary objectForKey:kJsonSchedulesKey];
-    NSMutableDictionary         *newDictionary = [dictionary mutableCopy];
-    NSMutableArray              *newSchedulesArray = [NSMutableArray new];
-    
+
+    NSArray              *schedules = [dictionary objectForKey:kJsonSchedulesKey];
+    NSMutableDictionary  *newDictionary = [dictionary mutableCopy];
+    NSMutableArray       *newSchedulesArray = [NSMutableArray new];
+
     for (NSDictionary *schedule in schedules) {
         [newSchedulesArray addObject:schedule];
     }
-    
-    [newDictionary setValue:[dictionary objectForKey:kJsonTasksKey]
-                     forKey:kJsonTasksKey];
-    
-    [newDictionary setValue:newSchedulesArray
-                     forKey:kJsonSchedulesKey];
-    
+
+    [newDictionary setValue:[dictionary objectForKey:kJsonTasksKey] forKey:kJsonTasksKey];
+
+    [newDictionary setValue:newSchedulesArray forKey:kJsonSchedulesKey];
+
```

Comment:
Why return a mutable copy NSDictionary? 
If a copy is needed, why just do `return [dictionary copy]`